### PR TITLE
Fixed Transaction Update

### PR DESCRIPTION
**Pull request by piyushandilya**

### CODE REVIEW

File: src/main/java/com/uptech/accounted/controller/TransactionController.java [hunk 1]
```diff
@@ -262,6 +272,15 @@ public String getNarration() {
     return narration.getText();
   }
 
+  public Long getId() {
+    try {
+      return Long.parseLong(id.getText());
+    } catch(Exception exception) {
+      exception.getMessage();
```
Comment (amanmanocha) [inline src/main/java/com/uptech/accounted/controller/TransactionController.java]: What is this?

Comment (amanmanocha) [reply src/main/java/com/uptech/accounted/controller/TransactionController.java]: Return small long and alert on exception.

File: src/main/java/com/uptech/accounted/controller/TransactionController.java [hunk 2]
```diff
@@ -296,6 +317,7 @@ public void initialize(URL location, ResourceBundle resources) {
         dateOfTransaction.setValue(selectedItem.getDateOfTransaction());
         amount.setText(selectedItem.getAmount().toString());
         narration.setText(selectedItem.getNarration().toString());
+        saveTransaction.setText("Update");
       } catch (NullPointerException nullPointerException) {
         nullPointerException.getMessage();
```
Comment (amanmanocha) [inline src/main/java/com/uptech/accounted/controller/TransactionController.java]: Alert here

Comment (piyushandilya) [reply src/main/java/com/uptech/accounted/controller/TransactionController.java]: Alert that "You will be overwriting the entry for ID= 13. Do you want to continue?". Like this?

**Pull request merged**